doc/conf.py

```diff
@@ -88,6 +88,13 @@ def _check_deps():
 
 autodoc_docstring_signature = True
 
+intersphinx_mapping = {
+  'python': ('http://docs.python.org/', None),
+  'numpy': ('http://docs.scipy.org/doc/numpy-dev/', None),
+  'scipy': ('http://scipy.github.io/devdocs/', None),
```

Comment:
Same as above, this should be the stable version of the docs (https://docs.scipy.org/doc/scipy/reference/)